packages/gestalt/src/TableOfContents.css

```diff
@@ -0,0 +1,12 @@
+.wrapper {
```

Comment:
this is a nit
container instead of wrapper? 

don't have to change, maybe i used wrapper myself in List...